Lavinia-api/Utilities/CsvFormats.cs

```diff
@@ -169,11 +169,11 @@ public ElectionTypeFormat Parse(string line, FieldParser parser)
     /// </summary>
     public class CountyDataFormat : ICsvFormat<CountyDataFormat>
     {
-        public double Areal;
-        public string County;
-        public int Population;
-        public int Seats;
-        public int Year;
+        public double Areal { get; private set; }
```

Comment:
This isn't really a problem with this commit, rather a problem from before -- in English it is called "Area", not "Areal"